packages/eddsa-ticket-pcd/src/EdDSATicketPCD.ts

```diff
@@ -46,53 +52,85 @@ export enum TicketCategory {
  * when verifying tickets.
  */
 export interface ITicketData {
-  // the fields below are not signed and are used for display purposes
-
+  // The fields below are not signed and are used for display purposes.
   attendeeName: string;
   attendeeEmail: string;
   eventName: string;
   ticketName: string;
   checkerEmail: string | undefined;
-
-  // the fields below are signed using the server's private eddsa key
-
-  ticketId: string; // primary key uuid of the ticket's `devconnect_pretix_tickets` entry
-  eventId: string; // primary key uuid of the event's `pretix_events_config` entry
-  productId: string; // primary key uuid of the ticket's `devconnect_pretix_items_info` entry
+  // The fields below are signed using the server's private eddsa key.
+  ticketId: string; // Primary key uuid of the ticket's `devconnect_pretix_tickets` entry.
+  eventId: string; // Primary key uuid of the event's `pretix_events_config` entry.
+  productId: string; // Primary key uuid of the ticket's `devconnect_pretix_items_info` entry.
   timestampConsumed: number;
   timestampSigned: number;
-  attendeeSemaphoreId: string; // stringified big int
+  attendeeSemaphoreId: string;
   isConsumed: boolean;
   isRevoked: boolean;
   ticketCategory: TicketCategory;
 }
 
+/**
+ * Interface containing the arguments that 3rd parties use to
+ * initialize this PCD package.
+ */
 export interface EdDSATicketPCDInitArgs {
   makeEncodedVerifyLink?: (encodedPCD: string) => string;
```

Comment:
worth describing what `makeEncodedVerifyLink` does. TL;DR it lets the PCD Card Body UI create a QR code from a pcd, which, when scanned, directs a scanner to a webpage that verifies whether this PCD is valid. as you may see, it's kind of a hack.

Reply:
I'll copy your TL;DR here, let me know if it's enough.